src/element/newElement.ts

```diff
@@ -290,6 +290,8 @@ export const getMaxContainerWidth = (container: ExcalidrawElement) => {
       return BOUND_TEXT_PADDING * 8 * 2;
     }
     return containerWidth;
+  } else if (container.type === "ellipse") {
+    return Math.round((width / 2) * Math.sqrt(2)) - BOUND_TEXT_PADDING * 2;
```

Comment:
can you add comment on what's the significance of multiplying by `1.41`? Seems a bit arbitary.